docs/conf/conf.py

```
@@ -1,11 +1,15 @@
+"""Sphinx configuration file for CLP documentation."""
+
+from datetime import datetime, timezone
 
 # -- Project information -------------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = "CLP"
 # NOTE: We don't include a period after "Inc" since the theme adds one already.
-copyright = f"2021-{date.today().year} YScope Inc"
+# Ignore A001: Shadowing built-in name 'copyright' for Sphinx config.
+copyright = f"2021-{datetime.now(tz=timezone.utc).year} YScope Inc"  # noqa: A001
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `A001`)

Remove unused `noqa` directive

(RUF100)